src/Selenium2Library/keywords/_tableelement.py

```diff
@@ -73,6 +73,37 @@ def table_cell_should_contain(self, table_locator, row, column, expected, loglev
             self.log_source(loglevel)
             raise AssertionError(message)
 
+    def table_cell_should_not_contain(self, table_locator, row, column, expected, loglevel='INFO'):
```

Comment:
I have kept all the arguments name same as other functions in the file to keep the consistency, if like I can change that.